modcc/astmanip.hpp

```diff
@@ -10,19 +10,30 @@
 
 // Create new local variable symbol and local declaration expression in current scope.
```

Comment:
I am sorry, but I find these comments confusing! You have a comment above a type definition, and it doesn't describe the type, and there is no comment directly above function definition. That is a recipe for confusion, and a recipe for comments that get stale after refactoring.

Why not have a one line comment above each type, something like `convenience type for returning local variable declarations`?